airflow/providers_manager.py

```diff
@@ -34,14 +34,14 @@
     import importlib.resources as importlib_resources
 except ImportError:
     # Try back-ported to PY<37 `importlib_resources`.
-    import importlib_resources
+    import importlib_resources  # noqa
```

Comment:
why do we need this now -- since it was not present earlier?

Reply:
The MyPy real-time scan integration complains about it in the IDE. Not sure why, but I guess it has something to do with the weird treatment of importlib_resources for different python versions. Adding # noqa is safe here and solves the problem:

![Screenshot from 2021-01-07 11-35-22](https://user-images.githubusercontent.com/595491/103882694-7b80d500-50dc-11eb-9c28-08a1012d2206.png)

It's my quest to have a green checkmark for all python files.

![Screenshot from 2021-01-07 11-36-19](https://user-images.githubusercontent.com/595491/103882773-95221c80-50dc-11eb-8c8a-192e903b1dbf.png)